internal/provider/vaultsecrets/data_source_vault_secrets_rotating_secret.go

```diff
@@ -23,7 +23,7 @@ type DataSourceVaultSecretsRotatingSecretModel struct {
 	AppName        types.String `tfsdk:"app_name"`
 	ProjectID      types.String `tfsdk:"project_id"`
 	OrgID          types.String `tfsdk:"organization_id"`
-	SecretName     types.String `tfsdk:"secret_name"`
+	Name           types.String `tfsdk:"name"`
```

Comment:
I think we should leave this as SecretName since this is part of the TF provider data source API. Changing it will break existing TF configs.